src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {

```

Comment:
Ensure that interfaces have a single purpose. If 'AreaCalculator' and 'Drawable' contain methods that are conceptually different, they should not be implemented by the same class.